app/models/customer.rb

```diff
@@ -16,6 +16,6 @@ def self.top_customers
   end
   
   def total_transactions
-    transactions.where(result: "success").count
+    transactions.where( result: "success").count
   end
```

Comment:
Is this method still being used?

Reply:
No we will remove it in the next branch!  Thanks for the catch!

Reply:
Nevermind, this is for customers, not merchant customers. Keep please!